pom.xml

```diff
@@ -190,6 +190,23 @@
                 <version>3.3.0</version>
             </plugin>
 
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.2</version>
+                <configuration>
+                    <archive>
+                        <manifestEntries>
+                            <Specification-Title>Liquibase</Specification-Title>
```

Comment:
You probably don't need the Specification-* elements. We're not really using that for anything so far so probably don't need to maintain it.

The rest are good

Reply:
I thought it might be useful in the future if there is a need to know which Liquibase core version maps to which Neo4j extension version